### PR TITLE
managing-servers solution

### PR DESCRIPTION
**Pull request by tkellen**
cc @MattSurabian

### CODE REVIEW

File: exercises/managing-servers/ansible/playbook.yml [hunk 1]
```diff
@@ -0,0 +1,75 @@
+- hosts: all
+  become: yes
+  become_method: sudo
+  vars:
+    base_path: /mnt
+    site_fdqn: # from ec2 instance tags
```
Comment (tkellen) [inline exercises/managing-servers/ansible/playbook.yml:7]: check to see if this is actually leaving these variables empty

File: exercises/managing-servers/ansible/templates/nginx.conf [hunk 2]
```diff
@@ -0,0 +1,11 @@
+upstream app {
+  server localhost:8000;
```
Comment (MattSurabian) [inline exercises/managing-servers/ansible/templates/nginx.conf]: Could: make an `app_port` variable or similar

File: exercises/managing-servers/overview.md [hunk 3]
```diff
@@ -6,14 +6,14 @@ Keypair
 Security Groups
 IAM roles
 
-
 ## EXERCISE
 
 ## LEARNING OBJECTIVES
 
-- What is a machine image used for?
+- How do you find machine images?
```
Comment (MattSurabian) [inline exercises/managing-servers/overview.md]: `Where` vs `How`? Could: Add something about IDs

File: exercises/managing-servers/terraform.tf [hunk 4]
```diff
@@ -0,0 +1,90 @@
+variable "name" { default = "learn-terraform" }
+variable "fqdn" { default = "learn-terraform.com" }
+variable "bucket" { default = "learn-terraform-bucket" }
```
Comment (MattSurabian) [inline exercises/managing-servers/terraform.tf:16]: This needs to be different for everyone

Comment (tkellen) [inline exercises/managing-servers/terraform.tf:16]: I'll add a comment for this

File: exercises/managing-servers/terraform.tf [hunk 5]
```diff
@@ -0,0 +1,90 @@
...
+#
+provider "aws" {
+  region = "us-east-1"
+  profile = "YOUR_PROFILE_NAME"
+}
+
+##
+# A S3 bucket for holding static data.
+#
+resource "aws_s3_bucket" "learn-terraform" {
```
Comment (MattSurabian) [inline exercises/managing-servers/terraform.tf]: Should: rename every resource `"main"` or something equally generic.

**Comment by tkellen** [comment]
ready for re-review

**Comment by MattSurabian** [comment]
:+1:

**Pull request merged**